js/plugin-loader.js

```diff
@@ -326,6 +326,10 @@ Plugin.prototype = {
             else {
               throw new Error (`No file name for data service`)
             }
+            // Make the relative path clear. process.cwd() is zlux-example-server/bin/
+            if (!path.isAbsolute(fileLocation)) {
+              fileLocation = path.join(process.cwd(),fileLocation);
```

Comment:
Can this even be right? Will a data service ever be found under just process.cwd(), which is, as you're saying, 'zlux-example-server/bin/' ?